tests/test_multi_ifdown.py

```diff
@@ -240,7 +243,7 @@ def test_015_finalize_failover_after_most_advanced_secondary_gets_back():
 
     # and, node2 should finally become the primary without losing any data
     print()
-    assert node2.wait_until_state(target_state="primary")
```

Comment:
Why is this change expected?

Reply:
We just do the same test later, waiting for the primary node first have an higher chance of reaching a timeout compared to waiting for the secondary nodes first.